flepimop/gempyor_pkg/src/gempyor/config_validator.py

```diff
@@ -331,13 +356,17 @@ def check_dates(cls, values):
         end_date = values.get('end_date')
         if start_date and end_date:
             if end_date <= start_date:
-                raise ValueError('end_date must be greater than start_date')
+                raise ValueError(
+                    f'`end_date` ({end_date}) must be greater than `start_date` ({start_date})).'
+                )
         return values
     
     @model_validator(mode='before')
     def init_or_seed(cls, values):
         init = values.get('initial_conditions')
         seed = values.get('seeding')
         if not init or seed:
-            raise ValueError('either initial_conditions or seeding must be provided')
+            raise ValueError(
+                f'At least one of `initial_conditions` or `seeding` must be provided.'
+                )
```

Comment:
is this "NOT(init) OR seed" or "NOT(init OR seed)"?

I had to look, it seems like its the first, and that isn't what we want (based on the error): https://stackoverflow.com/questions/16679272/priority-precedence-of-the-logical-operators-order-of-operations-for-not-an

Reply:
I moved this to GH-254. I think changing this is outside the scope of this PR and this code is not in active use.